const displayError = (a, b, data) => {
    console.log("animateElement arrow function called");
    window.location.reload();
    data.push(value);
    alert('Action completed!');
};